wikidatarefisland/pipes/scraper.py

```diff
@@ -0,0 +1,91 @@
+import concurrent.futures
+from time import gmtime, strftime
+
+import extruct
+import requests
+from w3lib.html import get_base_url
+
+from .abstract_pipe import AbstractPipe
+
+
+class ScaraperPipe(AbstractPipe):
+    def __init__(self, config, schemaorg_normalizer, schemaorg_mapper):
+        """
+
+        :type config: wikidatarefisland.Config
+        :type schemaorg_mapper: wikidatarefisland.services.SchemaorgPropertyMapper
+        """
+        self.config = config
+        self.schemaorg_normalizer = schemaorg_normalizer
+        self.schemaorg_mapper = schemaorg_mapper
+
+    def flow(self, item):
+        resource_urls = {i['url']: i['referenceMetadata'] for i in item['resourceUrls']}
+        schemaorg_mapping = {
+            i['url']: i['property'] for i in self.schemaorg_mapper.get_mapping()}
+        statement_pids = [i['pid'] for i in item['statements']]
+        properties_to_check = filter(lambda t: t in schemaorg_mapping.values(), statement_pids)
+        if not properties_to_check:
+            # Bail out if we can't find any property that have schema.org equivalent
+            return
+
+        extracted_data = {}
+        with concurrent.futures.ThreadPoolExecutor(max_workers=20) as executor:
```

Comment:
Should the worker count either be injected or read from the config?

Reply:
Done.